docs/Getting-Started/Create-bootable-media/bootable-usb-using-linux.md

```diff
@@ -0,0 +1,111 @@
+
+## III. Verify the Pop!_OS.iso File
+
+You should verify that your download matches the checksum before trying to install. This ensures that you've received the full, complete download and that it is not corrupted. Compare the hash values generated using the below commands to the values shown on the [Pop!_OS download page](https://pop.system76.com/). **Do not use the image below to verify hash values.**
+
+![Compare Hash Values](/images/create-bootable-usb-linux/compare-hash-values.png)
+
+When entering the appropriate checksum command below, ensure:
+
+- The file name in your command matches the downloaded ISO in your local directory. 
+- The examples below use the ~/Downloads folder. Make sure the appropriate explicit or relative path is used in your command.
+
+>**Note**: Popsicle also provides a feature for easily checking the SHA256 checksum of an ISO file (Step 3). Skip this section if you prefer a GUI method.
+### Verify the Intel/AMD ISO
+
+```bash
+sha256sum Downloads/pop-os_21.10_amd64_intel_7.iso
+```
+
+### Verify the NVIDIA ISO
+
+```bash
+sha256sum Downloads/pop-os_21.10_amd64_nvidia_7.iso
+```
+
+### Verify the RASP PI 4 ISO
+
+```bash
+sha256sum Downloads/pop-os_21.10_arm64_raspi_7.img.xz
+```
+
+## IV. Create the Bootable USB
+
+Flash your USB media with the Pop!_OS.iso image. Note that this process will completely erase all previous data on the USB storage device.
+
+1. Launch Popsicle. The app may display as USB Flasher in search results.
+
+![Launch Popsicle](/images/create-bootable-usb-linux/launch-popsicle.png)
+
+2. Click `Choose Image`.
+
+![Choose Image](/images/create-bootable-usb-linux/choose-image.png)
+
+3. Navigate to local directory with your Pop!_OS.iso file and select it.
+
+>**Note**: You can optionally check the hash of the ISO file in this step before clicking `Next`.
+
+![Find Image](/images/create-bootable-usb-linux/find-image.png)
+
+4. Click `Next`. 
+
+5. Select the USB installation media. Click `Next`.
+
+![Select Media](/images/create-bootable-usb-linux/select-media.png)
+
+6. Enter your system password if prompted; then wait for the installation to complete.
+
+![Flashing Process](/images/create-bootable-usb-linux/flashing-process.png)
+
+7. Click `Done` when the process completes. 
+
+## V. Next Steps
+
+Use your bootable Pop!_OS USB to demo, install, or recover a current Pop!_OS installation.
+
+### Install Pop!_OS
+
+Poweroff the target computer and insert the bootable USB. Power on the computer and enter the boot device menu selection for your BIOS or UEFI system. Follow the steps in the [Standard Installation](/Getting-Started/Installation/installation.md) section.
+
+>**Note**: Systems with Secure Boot enabled must disable this feature before installing Pop!_OS. See [Microsoft's official documentation](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/disabling-secure-boot?view=windows-10) for this procedure.
```

Comment:
Are we linking to Microsoft because we consider them an authority on Secure Boot (or because Secure Boot is "their fault?") Their article is pretty generic since they can't anticipate the user's exact brand and firmware any better than we can. I don't personally have a problem with linking this (the Microsoft article provides useful background information and explains the procedure just as well as we could), but it could be seen as a crutch since Pop!_OS is competing with Windows.

Reply:
Some System76 hardware that has a proprietary BIOS (like the AMD laptops and most desktops) support Secure Boot, so it's not completely a third-party Windows laptop thing. It's not a supported feature but it's there.

Reply:
Here's what I'd need help with if we added this section: 
- Explain why Pop!_OS doesn't like secure boot
- Mention that secure boot compatibility is on the horizon in the [development roadmap ](https://github.com/pop-os/iso/issues/284)(unless it's not)
- Add a disclaimer listing the possible concerns with disabling secure boot (malware)
- General steps (booting into BIOS/EUFI). We can use the MS documentation for this, unless there's a popular system (like a thinkpad) that would serve as a good example in our context.

Reply:
I think QA can better answer the first two. 

I don't think there's much increased risk for the average user by disabling secure boot. Linux is already more defensive than Windows. And I'm not knocking Windows here, they have a much larger attack surface to cover. But I would feel more comfortable disabling secure boot to install a Linux distro, than using a Windows machine with secure boot on.
We typically tell users who ask that they don't need to install antivirus software on Linux unless they're running a server or sending lots of files back and forth to Windows folks and they are concerned about passing something on.

The keys to boot into the BIOS are going to be different from every manufacturer, and even different per model. Usually it's one of the function keys, the DEL key or ESC. Users should check on a make/model basis. Also, many systems show keyboard shortcuts on the splash screen.

Reply:
I'll add this as a section under the Installation chapter. I'll wait for more feedback from SQA to determine if this new section should be a part of this PR.